components/Modal/index.js

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import ReactDOM from "react-dom";
+import {
+  ModalWrapper,
+  StyledModal,
+  ModalOverlay,
+  ModalBody,
+} from "./style";
+
+const Modal = ({ onClose, children }) => {
+  const handleCloseClick = (e) => {
+    e.preventDefault();
+    onClose();
+  };
+
+  const modalContent = (
+    <ModalOverlay>
+      <ModalWrapper>
+        <StyledModal>
+          <ModalBody>{children}</ModalBody>
+        </StyledModal>
+      </ModalWrapper>
+    </ModalOverlay>
+  );
+
+  return ReactDOM.createPortal(
```

Comment:
If we destructure above, then we can do the following here.

```suggestion
  return createPortal(
```